docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -162,6 +164,63 @@ fwupdmgr refresh
 fwupdmgr update
 ```
 
+## Dasharo EC Transition
+
+DTS allows to perform full EC transition from vendor to Dasharo/ec FW.
+
+To perform EC transition make sure you are running DTS version 1.0.1 or higher
+and follow this steps:
+
+* After boot, choose option number 9 to drop to Shell
+* Download BIOS and EC update files. You can download them from
+[there (BIOS)](https://cloud.3mdeb.com/index.php/f/491273) and
+[there (EC)](https://cloud.3mdeb.com/index.php/f/491331)
+> Note: At the time writing this instruction, current highest version is 1.3.0
+Always make sure you are downloading the newest version.
+
+* Run script for EC transition
+```bash
+ec_transition novacustom_ns5x_v1.3.0.rom novacustom_ns5x_v1.3.0_ec.rom
+<TBD>
+```
+> Note: Make sure you have provided 2 argumets:
+The first is path to BIOS update file and the second is path to EC update file
+
+* Computer will shut down automaticly
+* Power on your computer. Booting process may take a while
+* Your firmware is correcty installed. You can retrieve FW information using
+```bash
+system76_ectool info
+<TBD>
+```
+
+## Dasharo EC update
+
+DTS allows to update EC to newer version. To properly update it, follow this
+steps:
+
+* Retrieve information about your current EC
+```bash
+system76_ectool_info
+<TBD>
+```
+* Download EC update file. You can find it
+[there (EC)](https://cloud.3mdeb.com/index.php/f/491331)
```

Comment:
I think that at this point we should write something like this: Download the newest version of Embedded Controller firmware.

We should also not put hyperlinks here, but show the command to get the firmware.